internal/config/testdata/advanced.yml

```diff
@@ -22,7 +22,7 @@ log:
 
 cors:
   enabled: true
-  allowed_origins: "foo.com bar.com  baz.com"
```

Comment:
I think this was intentional to demonstrate that multiple whitespace tokens are treated as the entire delimiter so I would reinstate this as it was. Maybe we a comment if it is unclear.